data.proto

```diff
@@ -179,3 +181,66 @@ message PlayRunConfig {
   bool observer = 3;
   uint32 trial_count = 4;
 }
+
+message SelfPlayActorConfig{
+  uint32 num_input = 1;
+  uint32 num_input_2 = 2;
+  uint32 num_action = 3;
+  MLPNetworkConfig actor_network = 4;
+  MLPNetworkConfig critic_network = 5;
+  ModelArgs model_kwargs = 6;
+  string run_id = 7;
+  string model_id = 8;
+  int32 model_version = 9;
+  string environment_implementation = 10;
+  repeated float action_scale = 11;
+  repeated float action_bias = 12;
+  uint32 max_action = 13;
+  repeated uint32 alice_grid_shape = 14;
+  repeated uint32 bob_grid_shape = 15;
+}
+
+message SelfPlayRolloutConfig{
+  uint32 epoch_count = 1;
+  uint32 epoch_train_trial_count = 2;
+  uint32 epoch_test_trial_count = 3;
+  uint32 max_parallel_trials = 4;
+  uint32 model_publication_interval = 5;
+  uint32 number_turns_per_trial = 6;
+  uint32 test_freq = 7;
+}
+
+message SelfPlayReplaybufferConfig{
+  uint32 min_replay_buffer_size = 1;
```

Comment:
Why is minimum size required?

Reply:
I see that it's used to sample. But min_reply_buffer_size and batch_size shouldn't be 2 different parameters

Reply:
Any reason why? Those two could be different.

Reply:
The only place min_replay_buffer_size used was to ensure that the samples in it are larger than the batch size? (so that a full batch can be sampled)

Reply:
Yep, and also to ensure that there is enough variance in experience when the training starts. I see your point. I'll change the min_replay_buffer_size.